src/docs/perf/appendix.md

```diff
@@ -0,0 +1,156 @@
+When you want an animation to have a performance of 20 ms per frame or
+50 fps, there’s little room for different interpretations about the numbers. On
+the other hand, to describe the same animation in words, someone might call it
+good, while someone else might complain that it’s bad. Similarly, the same
+word or phrase could be interpreted differently by different people. You might
+interpret an OK frame rate to be 60 fps, while someone else might interpret it
+to be 30 fps.
+
+Numbers could still be noisy. For example, the measured time per frame might
+be a true computation time of this frame, plus a random amount of time (noise)
+that CPU/GPU spends on some unrelated work. Hence, the metric will fluctuate.
+Nevertheless, there’s no ambiguity of what the number means. And, there are
+also rigorous theory and testing tools to handle such noise. For example, you
+could take multiple measurements to estimate the distribution of a random
+variable, or you could take the average of many measurements to eliminate the
+noise by [the law of large numbers][1].
+
+### 3. Performance is comparable and convertible
+
+Performance numbers not only have unambiguous meanings, but they also have
+unambiguous comparisons. For example, there’s no doubt that 5 is greater than 4.
+On the other hand, it might be subjective to figure out whether excellent is
+better or worse than superb. Similarly, could you figure out whether epic is
+better than legendary? Actually, the phrase _strongly_ _exceeds_ _expectations_
+could be better than _superb_ in someone’s interpretation. It only becomes
+unambiguous and comparable after a definition that maps strongly exceeds
+expectations to 4 and superb to 5.
+
+Numbers are also easily convertible using formulas and functions. For example,
+60 fps can be converted to 16.67 ms per frame. A frame's rendering
+time _x_ (ms) can be converted to a binary indicator
+`isSmooth = [x <= 16] = (x <= 16 ? 1 :0)`. Such conversion can be compounded or
+chained, so you can get a large variety of quantities using a single
+measurement without any added noise or ambiguity. The converted quantity can
+then be used for further comparisons and consumption. Such conversions are
+almost impossible if you're dealing with natural languages.
+
+### 4. Performance is fair
+
+If issues rely on verbose words to be discovered, then an unfair advantage is
+given to people who are more verbose (more willing to chat or write) or those
+who are closer to the development team, who have a larger bandwidth and lower
+cost for chatting or face-to-face meetings.
+
+By having the same metrics to detect problems no matter how far away or how
+silent the users are, we can treat all issues fairly. That, in turn,
+allows us to focus on the right issues that have greater effects.
+
+### How to make performance useful
+
+The following is a summary the 4 points that were discussed, from a slightly
```

Comment:
=> The following summarizes the 4 points discussed here, from a slightly different perspective.  (Note that perspective was misspelled.)

Reply:
Done.